autoware_lint_common/README.md

```diff
@@ -22,6 +22,10 @@ if(BUILD_TESTING)
 endif()
 ```
 
+_For ROS 2 messages and services,_
+
+If there is `ADD_LINTER_TESTS` argument in the `rosidl_generate_interfaces()` function in the `CMakelists.txt` file, you need to remove the `ADD_LINTER_TESTS` argument.
+
```

Comment:
I personally this is enough. I'll leave it to you whether to apply it or not.

```suggestion
For ROS 2 messages and services, you need to remove the `ADD_LINTER_TESTS` argument in the `rosidl_generate_interfaces()` function in the `CMakelists.txt` file.

```

Reply:
Yeah, this is simpler and enough.